packages/spec/src/spec.emu.html

```diff
@@ -379,8 +379,11 @@ <h1>Syntactic Grammar</h1>
     TemplateParameterList `,` TemplateParameter
 
 TemplateParameter :
-    Identifier TemplateParameterDefault?
+    Identifier TemplateParameterConstraint? TemplateParameterDefault?
```

Comment:
There might be an issue with .tmlanguage here needing updating too. I think it will get masked by semantic coloring, though, so to observe it visually if it's there, you'd have to turn that off. If I'm reading right, we won't match extends as keyword in this position and not sure if anything else will get impacted.

Reply:
hhm yeah forgot to add test for that, will do that and fix if there is an issue